src/css/styles.css

```diff
@@ -57,10 +57,16 @@ p {
   font-weight: var(--t-font-weight-regular);
 }
 
+strong {
+  font-weight: var(--t-font-weight-semibold);
+}
+
```

Comment:
Don't use words like regular and semibold in variables, just --text-font-weight is good